azure-pipelines.yml

```
@@ -86,7 +86,7 @@ jobs:
   parameters:
     versions: ['3.8']
     images: ['ubuntu-18.04']
-    package: '-e .[tf]'
+    package: '-e .[tf, plt]'
```

The space is causing pip to treat `plt]` as a separate requirement.

```suggestion
    package: '-e .[tf,plt]'
```